0-ad-history-mod-package/simulation/templates/structures/athen/house.xml

```diff
@@ -3,6 +3,13 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Oikos</SpecificName>
+    <History>
+    Unfortunately, houses in Ancient Greece have not aged well, and usually the most archeologists can find is a foundation. While renown for their magnificent stone temples, the Greeks apparently skimped on their own houses. A house typically was constructed from mud bricks. Which wasn’t durable at all so much that burglars usually dug a hole in the wall rather than breaking through the wall.
```

Comment:
```suggestion
    Unfortunately, houses in Ancient Greece have not aged well, and usually the most archeologists can find is a foundation. While renown for their magnificent stone temples, the Greeks apparently skimped on their own houses. A house typically was constructed from mud bricks. Which wasn’t durable at all, so much that burglars usually dug a hole in the wall rather than breaking through the wall.
```